### DISCUSSION TITLE
VRM Temperature...

### DISCUSSION THREAD

**Discussion by Spuner** [Rem0o/FanControl.Releases#2854] [Q&A]
Which of these indicators is responsible for VRM?

![2024-11-28 07_50_32-Fan Control V208 (Profile 1 - Auto Fans Control json)](https://github.com/user-attachments/assets/9677708c-2157-465a-834f-26fe7eaddf4a)

**Comment by Fr0stX76** [comment]
I don't think any of those is the VRM temp. To find out, I would use HWinfo which has a more detailed description of sensors, then see what FC temp matches

Note that on many Asus MB, VRM reporting is done via the EC chip, so you might have to activate that option in both FC and HWinfo to get the reading.

Note2, some MB don't support VRM reading at all...

**Comment by Spuner** [comment]
In FC I activated EC, but nothing was added to the list. In HWinfo the sensor shows the temperature...

![2024-12-01 20_27_41-Greenshot](https://github.com/user-attachments/assets/87384ceb-e4b7-405b-9bc9-21328723bef2)

**Comment by Fr0stX76** [comment]
Make sure you restart FC after enabling EC in it.

If not, then it most likely is because this specific Asus embbeded controller is not suppoted by LibreHardwareMonitor.

To confirm, simply run LibreHardwareMonitor and see if the sensor shows in it

https://github.com/LibreHardwareMonitor/LibreHardwareMonitor/releases

 A workaround would be to use the HWinfo pluggin in FC

https://github.com/Rem0o/FanControl.HWInfo

**Comment by Spuner** [comment]
Yes, indeed, LibreHardwareMonitor does not display VRM temperature. It worked through the plugin. Thank you very much for your help )

**Comment by Spuner** [comment]
I noticed that if I close the HWinfo application, the plugin no longer works...

**Comment by Fr0stX76** [comment]
You have to keep it running, it feeds FC... There is no workaround to this

But you can have it close to tray/ open with Windows to tray and use only the sensor panel

**Comment by Spuner** [comment]
The problem is that the plugin only works when the sensor window is open, and without the sensor window it cannot start automatically, it must be started manually. Because of this, for me the whole point of this plugin is lost, unfortunately.

**Reply by Rem0o** [reply]
Sensor window doesn't need to be visible/on, as long as HWInfo is running in sensor mode.

**Reply by Spuner** [reply]
I understand what you mean. But here comes another problem... If FC starts before HWinfo, then this method will not work, because after that you have to manually close/open HWinfo, then update the sensors in FC, so that FC sees the plugin. Yes, this would work fine if HWinfo started before FC.

**Reply by Fr0stX76** [reply]
Increase the delay to FC

Settings/general/Start at user logon delay

Also, disabling some sensors you din't need in HWinfo will help decrease it's startup time

**Reply by Spuner** [reply]
I set the delay to 5 seconds. Let's see...
